src/volume_mgr.rs

```diff
@@ -114,7 +143,11 @@ where
     ///
     /// This function gives you a `RawVolume` and you must close the volume by
     /// calling `VolumeManager::close_volume`.
-    pub fn open_raw_volume(&mut self, volume_idx: VolumeIdx) -> Result<RawVolume, Error<D::Error>> {
+    pub fn open_raw_volume(
+        &mut self,
+        volume_idx: VolumeIdx,
+        read_only: bool,
```

Comment:
I think I'd prefer an `enum VolumeOpenMode { ReadOnly, ReadWrite }`, otherwise the caller writes:

`thing.open_raw_volume(VolumeIdx(0), true)`

and it's totally unclear what `true` means from the context.

Reply:
Good point thanks!

Reply:
changed in 2fd18cbaa92ca31ceac2922de05e50e3511ca88b